packages/datetime/test/dateInputTests.tsx

```diff
@@ -92,7 +92,11 @@ describe("<DateInput>", () => {
         assert.isFalse(wrapper.find(Popover).prop("isOpen"));
     });
 
-    it("Popover closes when tabbing on first day of the month", () => {
+    // Skipping because simulate just invokes the function passed to React's "on<EventName>" prop
+    // and doesn't actually simulate anything. Properly testing would require running with an actual
+    // browser and focusing specific elements via the DOM API. This would require changing the Karma
+    // config to run with Chrome instead of ChromeHeadless.
+    it.skip("Popover closes when tabbing on first day of the month", () => {
```

Comment:
I do have a working local version but could only get it to work by switching the Karma browser from `ChromeHeadless` to `Chrome`. The problem with simulate is that there's no way to get the `focusin` event handler functions to run because React doesn't provide an `onFocusIn` prop, only `onFocus`. If I use the DOM API instead and call `Element#focus()`, the browser will fire a `focusin` event on my behalf.

Reply:
ok, this is fine to skip for now. testing focus behavior is tricky in these test suites